Formula/o/openjdk@8.rb

```diff
@@ -32,6 +32,10 @@ class OpenjdkAT8 < Formula
   uses_from_macos "unzip"
   uses_from_macos "zip"
 
+  on_ventura :or_newer do
+    disable! date: "2024-01-25", because: "is incompatible with Xcode 15+"
+  end
```

Comment:
What is exact issue? Do you have OpenJDK bug id?

You also mentioned OpenJDK 11 but builds/runs with upstream backport https://github.com/openjdk/jdk11u-dev/commit/dc028f28d1ec5a4efd89af1b5f83fa4dc349defc for https://bugs.openjdk.org/browse/JDK-8323008 (marked as affecting only 11, 17, 21, 22, 23).

---

Anyway, using `disable!` would be disruptive for users as it blocks installing the older bottle. Currently, Sonoma users are running off of the Ventura-built bottle, but this line would prevent installation.

Reply:
That was just an experiment. Disregard it: this is still a draft PR.